packages/components/.eslintrc.json

```diff
@@ -0,0 +1,76 @@
+{
+  "env": {
+    "browser": true,
+    "es2021": true
+  },
+  "extends": [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:@stencil-community/recommended"
+  ],
+  "overrides": [
+  ],
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "ecmaVersion": "latest",
+    "sourceType": "module",
+    "project": "./tsconfig.json"
+  },
+  "plugins": [
+    "@typescript-eslint"
+  ],
+  "rules": {
+    "indent": [
+      "error",
+      2,
+      {
+        "SwitchCase": 1
+      }
+    ],
+    "quotes": [
+      "error",
+      "single"
+    ],
+    "semi": [
+      "error",
+      "always"
+    ],
+    "@typescript-eslint/no-unused-vars": [
+      "error",
+      {
+        "caughtErrors": "none",
+        "destructuredArrayIgnorePattern": "^_",
+        "ignoreRestSiblings": true
+      }
+    ],
+    "@stencil-community/strict-boolean-conditions": "off",
+    "@stencil-community/required-prefix": ["error", ["post"]],
+    "@stencil-community/async-methods": "error",
+    "@stencil-community/ban-prefix": ["error", ["stencil", "stnl", "st"]],
+    "@stencil-community/decorators-context": "error",
+    "@stencil-community/decorators-style": [
+      "error", {
+        "prop": "inline",
+        "state": "inline",
+        "element": "inline",
+        "event": "inline",
+        "method": "multiline",
+        "watch": "multiline",
+        "listen": "multiline"
+      }],
+    "@stencil-community/element-type": "error",
+    "@stencil-community/host-data-deprecated": "error",
+    "@stencil-community/methods-must-be-public": "error",
+    "@stencil-community/no-unused-watch": "error",
+    "@stencil-community/own-methods-must-be-private": "error",
+    "@stencil-community/own-props-must-be-private": "error",
+    "@stencil-community/prefer-vdom-listener": "error",
+    "@stencil-community/props-must-be-public": "error",
+    "@stencil-community/props-must-be-readonly": "error",
+    "@stencil-community/render-returns-host": "error",
+    "@stencil-community/required-jsdoc": "error",
+    "@stencil-community/reserved-member-names": "error",
+    "@stencil-community/single-export": "error",
+    "@stencil-community/strict-mutable": "error"
```

Comment:
Why did you override these values, are they not defined by @stencil-community/recommended?

Reply:
According to @stencil-community/stencil-eslint README, these are all [recommended rules](https://github.com/stencil-community/stencil-eslint#recommended-rules). However, using @stencil-community/recommended alone, I noticed that most of these errors were actually not shown and I had to add them explicitly to get them to work.

Reply:
Ah, I see. The standard rules could be an installable package as well... I think it's fine like this for now.